forge-gui/res/cardsfolder/upcoming/emergency_weld.txt

```diff
@@ -0,0 +1,7 @@
+Name:Emergency Weld
+ManaCost:1 B
+Types:Sorcery
+A:SP$ ChangeZone | Origin$ Graveyard | Destination$ Hand | TgtPrompt$ Select target artifact or creature card in your graveyard | ValidTgts$ Artifact.YouOwn,Creature.YouOwn | SubAbility$ DBToken | SpellDescription$ Return target artifact or creature card from your graveyard to your hand. Create a 1/1 colorless Soldier artifact creature token.
```

Comment:
Make the second sentence of this SpellDescription the SpellDescription for line 5